cob_cartesian_controller/CMakeLists.txt

```diff
@@ -4,16 +4,16 @@ project(cob_cartesian_controller)
 find_package(catkin REQUIRED COMPONENTS cmake_modules cob_srvs geometry_msgs roscpp roslib std_msgs std_srvs tf visualization_msgs)
```

Comment:
`cmake_modules` is `find_package`'d, but the respective lib `TinyXML` is not. Do we need it then (i.e. the `cmake_modules` part? Or do we need to `find_package` `TinyXML`?